src/components/pages/Offers/Offer/OfferPreviewLink/OfferPreviewLink.jsx

```diff
@@ -5,16 +5,18 @@
 import PropTypes from 'prop-types'
 import React, { useCallback } from 'react'
 
-import { WEBAPP_URL } from 'utils/config'
+import { WEBAPP_URL_NEW, WEBAPP_URL_OLD } from 'utils/config'
 
-const getOfferPreviewUrl = (offerId, mediationId) => {
-  const webappOfferUrl = `${WEBAPP_URL}/offre/details/${offerId}`
+const getOfferPreviewUrl = (offerId, mediationId, webAppV2Enabled) => {
+
+  let webappOfferUrl = ''
+  webAppV2Enabled === true ?  webappOfferUrl = `${WEBAPP_URL_NEW}/offre/details/${offerId}` : webappOfferUrl = `${WEBAPP_URL_OLD}/offre/details/${offerId}`
 
   return mediationId ? `${webappOfferUrl}/${mediationId}` : webappOfferUrl
 }
 
-const OfferPreviewLink = ({ offerId, mediationId }) => {
-  const offerPreviewUrl = getOfferPreviewUrl(offerId, mediationId)
+const OfferPreviewLink = ({ offerId, mediationId, webAppV2Enabled }) => {
```

Comment:
Au lieu de passer par le container d'`OfferDetails` on vas préférer utiliser le hook `useActiveFeature` directement ici.
```suggestion
import useActiveFeature from 'components/hooks/useActiveFeature'
// ...
const OfferPreviewLink = ({ offerId, mediationId, webAppV2Enabled }) => {
    const isWebappV2Enabled = useActiveFeature('WEBAPP_V2_ENABLED') 
    const baseUrl = isWebappV2Enabled ?  WEBAPP_V2_URL_NEW : WEBAPP_URL
    const offerPreviewUrl = getOfferPreviewUrl(offerId, mediationId, baseUrl)
```